pkg/kubectl/cmd/create_configmap.go

```diff
@@ -55,7 +60,7 @@ var (
 // ConfigMap is a command to ease creating ConfigMaps.
 func NewCmdCreateConfigMap(f *cmdutil.Factory, cmdOut io.Writer) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "configmap NAME [--from-file=[key=]source] [--from-literal=key1=value1] [--dry-run]",
+		Use:     "configmap NAME [--from-file=[key=]source] [--from-binary-file=[key=]source] [--from-literal=key1=value1] [--dry-run]",
```

Comment:
also not sure how well binary content would plumb through a command line literal